service/src/main/java/org/mskcc/cmo/metadb/service/impl/MetaDbRequestServiceImpl.java

```diff
@@ -73,31 +73,68 @@ public boolean saveRequest(MetaDbRequest request) throws Exception {
                 request.setMetaDbSampleList(updatedSamples);
             }
             requestRepository.save(request);
-            return true;
+            return Boolean.TRUE;
         } else {
```

Comment:
How does it handle updatedRequestMetadata where the nats message is just the metadata json?